Cutelyst/Plugins/Utils/Validator/validator_p.h

```diff
@@ -29,39 +29,48 @@ namespace Cutelyst {
 class ValidatorPrivate
 {
 public:
-    ValidatorPrivate() :
-        stopOnFirstError(false)
-    {}
-
-    ValidatorPrivate(const ParamsMultiMap &parameters) :
-        params(parameters),
+    ValidatorPrivate(Context *c) :
         stopOnFirstError(false),
-        context(nullptr)
-    {}
+        context(c)
+    {
+        if (context) {
+            params = c->request()->parameters();
+        }
+    }
 
-    ValidatorPrivate(Context *c) :
+#ifdef Q_COMPILER_INITIALIZER_LISTS
+    ValidatorPrivate(Context *c, std::initializer_list<ValidatorRule*> list) :
         stopOnFirstError(false),
+        validators(list),
         context(c)
     {
-        params = c->request()->parameters();
+        if (context) {
+            params = c->request()->parameters();
+        }
     }
+#endif
+
+    ValidatorPrivate(const ParamsMultiMap &parameters) :
+        params(parameters),
+        stopOnFirstError(false)
+    {}
 
     ParamsMultiMap params;
     bool stopOnFirstError;
```

Comment:
if bool stopOnFirstError; is always false you could declare as
bool stopOnFirstError = false;
has the benefit of not forgetting to initialize if you add a new constructor